src/GLUniforms.jl

```diff
@@ -142,6 +145,7 @@ gl_promote(x::Type{Union{UInt16, UInt8}})  = x
 
 gl_promote{T <: AbstractFloat}(x::Type{T}) = Float32
 gl_promote(x::Type{Float16})               = x
+gl_promote(x::Type{Float64})               = x
```

Comment:
`gl_promote` is used to promote to the most common OpenGL type... I'd argue that `Float64` may be used, but still isn't the most common type... So I'd leave this out ;)